### ISSUE TITLE
Support connection without spanner.databases.get IAM permission

### ISSUE THREAD

**Issue by O1O1O1O** [yfuruyama/spanner-cli#10]
I have a service account that can't list databases but it would be nice to connect to a specific database if I know it by name and do stuff.  I'm able to do that with the SQLWorkbench/J and the Simba JDBC driver for Spanner.  

This is the error I get:
```
$ spanner-cli -p foo -i conversion -d conversion-prod -e 'select count(*) from account_owner'
2018/11/08 15:47:08 failed to connect to Spanner: rpc error: code = PermissionDenied desc = Resource projects/foo/instances/conversion/databases/conversion-prod is missing IAM permission: spanner.databases.get.
```

**Comment by yfuruyama** [comment]
That makes sense.
When `spanner-cli` launch, it will check the existence of database even though the credential doesn't have permission for it. (https://github.com/yfuruyama/spanner-cli/blob/master/session.go#L62)

I'll fix it by skipping that database existence check.

**Comment by O1O1O1O** [comment]
Maybe if the table is specified on the command line you *should* check it, but it should be possible to do a query without specifying the table on the command line since the table is always named in a `WHERE` clause.  So perhaps make database optional?   

Also remember I can do `SELECT * FROM database_name.table_name` so even connecting without an explicit database would make sense - but perhaps not practical.

**Comment by yfuruyama** [comment]
@O1O1O1O 
I fixed it by replacing `DatabaseGet` call with running `SELECT 1` query to check database existence, which doesn't require `spanner.databases.get` IAM permission any more (https://github.com/yfuruyama/spanner-cli/pull/11).
Also now the check only runs when `spanner-cli` launches as the interactive mode.

I have released `v0.2.0` including that fix, so please install and try again.
https://github.com/yfuruyama/spanner-cli/releases

Enjoy!

**Comment by O1O1O1O** [comment]
Thanks for the quick fix Furuyama-san. I've tested the new version and it is working great!
